src/Components/BehaviorsModal/Internal/Widgets/DataHistoryWidget/DataHistoryWidget.types.ts

```diff
@@ -31,6 +36,17 @@ export interface IDataHistoryWidgetSubComponentStyles {
     quickTimePicker: {
         dropdown?: IStyle;
         menuItemIcon?: CSSProperties;
-        calloutWidth?: number;
     };
+    errorContainer: {
+        root: CSSProperties;
+        image: Partial<IImageStyles>;
+        textContainer: IStyle;
+        descriptionContainer: IStyle;
+    };
+}
+
+export enum DataHistoryErrors {
```

Comment:
can we call these "error codes" `DataHistoryServiceErrorCodes` maybe?

Reply:
they probably belong in the constants file too rather than a types file.

Reply:
That assumes these are used to look up the locale string for the corresponding error from the service.

Reply:
I changed the name, but will keep it here instead of root constants file, since this enum is only used by this component.